src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -389,7 +396,19 @@
 # TODO: bindu update the help and samples after we finalize the commands.
 helps['acr build'] = """
     type: command
-    short-summary: Queues a new build based on the specified parameters.
+    short-summary: Queues a quick docker build providing interactive feedback.
+    examples:
+        - name: Queue a local context, pushed to ACR with streaming logs.
+          text: >
+            az acr build -t sample/helloworld:{{.Build.ID}} -r MyRegistry -c .
+    examples:
+        - name: Queue a local context, pushed to ACR without streaming logs.
+          text: >
+            az acr build -t sample/helloworld:{{.Build.ID}} -r MyRegistry --no-logs -c .
+    examples:
+        - name: Queue a local context, validating the build is successful, without pushing to the registry.
+          text: >
+            az acr build -r MyRegistry -c .
```

Comment:
We default pushing to registry to `true` so without pushing needs a parameter `--push-enabled false`.